modules/gcloud/gcloud.go

```diff
@@ -67,8 +70,26 @@ func (o Option) Customize(*testcontainers.GenericContainerRequest) error {
 
 // WithProjectID sets the project ID for the GCloud container.
 func WithProjectID(projectID string) Option {
-	return func(o *options) {
+	return func(o *options) error {
 		o.ProjectID = projectID
+		return nil
+	}
+}
+
+// WithDataYAML seeds the Bigquery project for the GCloud container with an io.Reader representing
```

Comment:
suggestion: brace to type so it doc links and use the canonical name
```suggestion
// WithDataYAML seeds the BigQuery project for the GCloud container with an [io.Reader] representing
```